midealocal/cloud.py

```diff
@@ -191,6 +191,13 @@ async def get_keys(self, appliance_id: int) -> dict[int, dict[str, Any]]:
         result.update(default_keys)
         return result
 
+    async def get_cloud_servers(self) -> dict[int, str]:

```

Comment:
This could be static,
```suggestion
    @staticmethod
    async def get_cloud_servers(self) -> dict[int, str]:
```